server/form-pages/apply/risk-and-need-factors/further-considerations/contingencyPlanQuestions.ts

```diff
@@ -61,7 +62,25 @@ export default class ContingencyPlanQuestions implements TasklistPage {
 
   questions: ContingencyPlanQuestionsRecord = questions
 
-  constructor(public body: ContingencyPlanQuestionsBody, private readonly application: Application) {}
+  contingencyPlanPartnerNames: Array<string> | null | Record<string, string>
+
+  constructor(public body: ContingencyPlanQuestionsBody, private readonly application: Application) {
+    let contingencyPlanPartnersNames = retrieveQuestionResponseFromApplication(
```

Comment:
Once the PR is merged I will use the following function for this: https://github.com/ministryofjustice/hmpps-approved-premises-ui/pull/580/commits/a57fd07ac3f3f1eb9ec237439fd204d5cc557792